### PR TITLE
Feature/juan pulido/system of lives

### PR DESCRIPTION
**Pull request by JPulido01**

### CODE REVIEW

File: Assets/Scripts/Movement.cs
Comment (Loko24) [inline Assets/Scripts/Movement.cs:1]: Just ask, where is located the logic about rest lives?

Comment (JPulido01) [reply Assets/Scripts/Movement.cs:1]: On lines 157 to 173 in the movement script

Comment (JPulido01) [reply Assets/Scripts/Movement.cs:1]: Already

**Review by Nik-Stevens — APPROVED** [review]
Approved

**Review by daniel374 — APPROVED** [review]

**Review by Loko24 — APPROVED** [review]

**Pull request merged**